packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -18,7 +18,11 @@ import type {
 } from 'react-reconciler/src/ReactTestSelectors';
 import type {ReactScopeInstance} from 'shared/ReactTypes';
 import type {AncestorInfoDev} from './validateDOMNesting';
+import type {FormStatus} from 'react-dom/src/ReactDOMFormActions';
 
+// TODO: For some reason the build script fails if this file doesn't include the
+// extension. I do not know why.
+import {NotPending} from 'react-dom/src/ReactDOMFormActions.js';
```

Comment:
@sebmarkbage Should ReactDOMFormActions.js live in react-dom-bindings instead of react-dom? I still don't fully understand what the boundary is between those packages.